accumulators/src/mpi.cpp

```diff
@@ -265,7 +265,7 @@
                 template void reduce(const communicator &, std::vector<std::vector<T>> const&, std::plus<T>, int);                                \
                 template void reduce(const communicator &, std::vector<std::vector<T>> const&, std::vector<std::vector<T>>&, std::plus<T>, int);
 
-            ALPS_INST_MPI_REDUCE(unsigned long)
+            ALPS_INST_MPI_REDUCE(std::size_t)
```

Comment:
with `clang` on `macos` this needs to be `unsigned long long` to avoid linker errors.